pioneer/packages/apps-config/src/settings/endpoints.ts

```diff
@@ -31,30 +31,35 @@ function createLive (t: TFunction): LinkOption[] {
   return [
     {
       info: 'joystream',
-      text: t<string>('rpc.joystream-1', 'Joystream (Europe/Germany - High Availabitliy)', { ns: 'apps-config' }),
-      value: 'wss://rome-rpc-endpoint.joystream.org:9944'
-    },
-    {
-      info: 'joystream',
-      text: t<string>('rpc.joystream-2', 'Joystream (JoystreamStats.Live)', { ns: 'apps-config' }),
-      value: 'wss://joystreamstats.live:9945'
-    },
-    {
-      info: 'joystream',
-      text: t<string>('rpc.joystream-3', 'Joystream (Europe/UK)', { ns: 'apps-config' }),
-      value: 'wss://testnet-rpc-3-uk.joystream.org'
-    },
-    {
-      info: 'joystream',
-      text: t<string>('rpc.joystream-4', 'Joystream (US/East)', { ns: 'apps-config' }),
-      value: 'wss://testnet-rpc-1-us.joystream.org'
+      text: t<string>('rpc.joystream-1', 'Joystream (Europe/Germany)', { ns: 'apps-config' }),
+      value: 'wss://rpc.joystream.org:9944/'
     },
     {
       info: 'joystream',
-      text: t<string>('rpc.joystream-5', 'Joystream (Singapore)', { ns: 'apps-config' }),
-      value: 'wss://testnet-rpc-2-singapore.joystream.org'
+      text: t<string>('rpc.joystream-2', 'Joystream (Old Giza Network)', { ns: 'apps-config' }),
+      value: 'wss://rome-rpc-endpoint.joystream.org:9944'
     }
     // {
+    //   info: 'joystream',
+    //   text: t<string>('rpc.joystream-2', 'Joystream (JoystreamStats.Live)', { ns: 'apps-config' }),
```

Comment:
This load balanced endpoint will stay available and https://pioneer.joystreamstats.live/ is going to be updated too,